requirements.txt

```diff
@@ -1,3 +1,3 @@
 flask
 Flask-Assets
-pyscss
+Flask-Webpack
```

Comment:
I was messing around with webpack earlier today >_< but this shouldn't be here I'll remove it tomorrow